instruments/abstract_instruments/instrument.py

```diff
@@ -144,6 +144,18 @@ def read(self, size=-1, encoding="utf-8"):
         """
         return self._file.read(size, encoding)
 
+    def read_raw(self, size=-1):
```

Comment:
Added this routine since I need to read the instrument raw - it sometimes sends back a bad header, see query and comments there. I figured that it is ugly to directly address `self._file.read_raw()` in the PMC8742 class, therefore added this into `instruments.py`.